demo/index.html

```diff
@@ -27,36 +53,62 @@
   </section>
 
   <section>
-    <h3>Setting items</h3>
-    <p>The data displayed by <code>vaadin-combo-box</code> can be set and accessed using the
-    <code>items</code> property. Only an array of <code>String</code> values is currently supported.
+    <h3>Setting items and label</h3>
+    <p>
+      The data items displayed by <code>vaadin-combo-box</code> can be set and accessed using the
+      <code>items</code> property. Only an array of <code>String</code> values is currently supported.
+      The <code>items</code> may be assigned with Polymer data-binding, using an attribute or the
```

Comment:
Any reason to limit data-binding to just Polymer? You can bind to the property using other frameworks also, right?